### PR TITLE
5883 managed object getbeans

### PR DESCRIPTION
**Pull request by benjamin-confino**

### CODE REVIEW

File: dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/managedobject/AbstractManagedObjectFactory.java [hunk 1]
```diff
@@ -276,6 +267,18 @@ public CDIRuntime getCDIRuntime() {
...
+        Bean<T> bean = null;
+        if (! nonContextual) {
+            bean = getBean();
+        }
+        String beanScope = null;
+        if (bean != null) {
+            beanScope = bean.getScope().getCanonicalName();
+        }
+        return beanScope;
+    }
```
Comment (tevans78) [inline dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/managedobject/AbstractManagedObjectFactory.java]: Needs comments but also how about structuring it like this...
```
    private String getBeanScope(boolean nonContextual) throws ManagedObjectException {
        String beanScope = null;
        if (! nonContextual) { //if nonContextual==true then there is no bean and no beanScope
            Bean<T> bean = getBean();
            if (bean != null) {
                beanScope = bean.getScope().getCanonicalName();
            }
        }
        return beanScope;
    }
```

Comment (benjamin-confino) [reply dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/managedobject/AbstractManagedObjectFactory.java]: Sure, why not

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build 

Previous build failed with a unit test issue on an unrelated project

**Comment by benjamin-confino** [comment]
All CDI and injection related tests passed.

**Pull request merged**